### PR TITLE
don't print the iTerm ansi escape when no TTY or on CI

### LINKED ISSUE #9: Bad output on Travis:
The iTerm special character prints on Travis:

https://travis-ci.org/sindresorhus/execa/jobs/126950278#L921

### PR DESCRIPTION
**Pull request by sindresorhus**
fixes #9

### CODE REVIEW

**Comment by sindresorhus** [comment]
@jamestalmage Could you review?

File: index.js [hunk 1]
```diff
@@ -73,8 +73,10 @@ module.exports = function (results) {
 
 	var output = '\n';
 
-	// make relative paths Cmd+click'able in iTerm
-	output += ansiEscapes.iTerm.setCwd();
+	if (process.stdout.isTTY && !process.env.CI) {
```
Comment (jamestalmage) [inline index.js]: Should we use `is-ci`? Does this method of detection miss anything?

**Comment by jamestalmage** [comment]
LGTM

File: index.js (hunk 1 above)
Comment (sindresorhus) [inline index.js]: Nah, not worth it. The CI's we care about uses this environment variable.

**Pull request merged**